server/src/main/resources/application.properties

```diff
@@ -9,7 +9,7 @@ spring.datasource.username=postgres
 spring.datasource.password=admin
 
 # Strategy how server works with database
-spring.jpa.hibernate.ddl-auto=update
+spring.jpa.hibernate.ddl-auto=create
```

Comment:
This changes destroy all stored data after reloading backend. E.g. I do my feature task, add some testing data, add some changes to Spring application and restart spring application. And all my data in databases dissappires